app/assets/stylesheets/govuk_publishing_components/components/_intervention.scss

```diff
@@ -0,0 +1,39 @@
+.gem-c-intervention {
+  @include govuk-text-colour;
+  @include govuk-responsive-padding(4);
+  @include govuk-responsive-margin(8, "bottom");
+
+  clear: both;
+  background-color: govuk-colour("light-grey", $legacy: "grey-4");
+
+  .govuk-govspeak {
+    p:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  .govuk-body:last-child {
+    margin-bottom: 0;
+  }
+}
+
+.gem-c-intervention__title {
+  @include govuk-font(24, $weight: bold);
+  margin-top: 0;
+  @include govuk-responsive-margin(4, "bottom");
+
+  &:last-child {
```

Comment:
There's only one instance of this element in the component, so is the `last-child` selector needed?

Reply:
removed